lib/std/zig/render.zig

```diff
@@ -37,10 +41,14 @@ pub const Fixups = struct {
     /// with this path.
     rebase_imported_paths: ?[]const u8 = null,
 
+    /// Temporary storage for captures to be discarded in the block immediately following them.
+    block_discards: []const Ast.TokenIndex = &.{},
```

Comment:
temporary storage is available on the Render struct. you don't need to abuse this Fixups struct.

Reply:
Thanks. I moved it.